src/panels/config/customize/ha-form-customize.js

```diff
@@ -26,9 +26,10 @@ class HaFormCustomize extends PolymerElement {
         if="[[computeShowWarning(localConfig, globalConfig)]]"
       >
         <div class="warning">
-          It seems that your configuration.yaml doesn't properly include
-          customize.yaml<br />
-          Changes made here won't affect your configuration.
+          It seems that your configuration.yaml doesn't properly
+          <a href="https://www.home-assistant.io/docs/configuration/customizing-devices/#customization-using-the-ui" target="_blank">include customize.yaml</a>.<br />
+          Changes made here are written in it, but will not be applied after a
```

Comment:
remove the `after a restart` it is not necessary to do a restart when making changes through the UI. (and there is a service for reloading customize `homeassistant.reload_core_config`)

Reply:
Removing that will send the wrong message. Pretty much the whole point of this change is that changes **are** applied immediately. But I'll change that to "after a configuration reload".

Reply:
Sorry missed the part about current run time, that would actually be a bug I think...

Reply:
I think it's fine as is, just the warning message needs a tweak.